<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperWindow</class>
 <widget class="QMainWindow" name="SuperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border : 1px solid black;
}

#centralWidget{
	background-image : url(:/Image/ChessBoard.jpg);
}

#Restart, #Undo, #Redo, #AI{
	border : none;
	border-radius : 5px;
 	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 rgb(0, 0, 255), stop:1 rgb(0, 0, 155));
	color : white;
	font-family : 'Consolas';
}

#Restart:hover, #Undo:hover, #Redo:hover, #AI:hover{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 rgb(0, 0, 200), stop:1 rgb(0, 0, 100));
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ChessBoardLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>320</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="BlackNumber">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>480</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLCDNumber" name="WhiteNumber">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="BlackPieceImage">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="WhitePieceImage">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Undo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Redo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AI">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BlackAI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="WhiteAI">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Timer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Outline</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
